h/static/scripts/search-client.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+var EventEmitter = require('tiny-emitter');
+var inherits = require('inherits');
+
+/**
+ * Client for the Hypothesis search API.
+ *
+ * SearchClient handles paging through results, canceling search etc.
+ *
+ * @param {Object} resource - ngResource class instance for the /search API
+ * @param {Object} opts - Search options
+ * @constructor
+ */
+function SearchClient(resource, opts) {
+  opts = opts || {};
+
+  var DEFAULT_CHUNK_SIZE = 200;
```

Comment:
This has increased from 100 to 200 -- that is possibly ok, but it might be better to leave it as is for now because of [the rather ugly way replies are loaded on the server](https://github.com/hypothesis/h/blob/3e8d22814a4e3ce8e6e743d5ca1ae59820c5dbe0/h/api/search/core.py#L58).